CHANGELOG.md

```diff
@@ -4,6 +4,16 @@
 
 - "You miss 100 percent of the chances you don't take. — Wayne Gretzky" — Michael Scott
 
+## 6.17.0
```

Comment:
Can we make this a patch version instead? This will allow the max amount of people to upgrade automatically - and there are no big features we are introducing in this version.

`feat(tracing)` changes are just adding tags to stuff, no big behaviour changes or improvements.

Reply:
I consider the Next.js v12 support a big feature. Do you think it should go into a patch?

Reply:
I agree with @AbhiPrasad here. Supporting it is a big feature, yet this is not much about supporting and more about enabling. 
Supporting it was something expected from us already.